service/src/main/java/com/ericsson/eiffel/remrem/generate/controller/RemremGenerateController.java

```diff
@@ -141,7 +145,7 @@ private JsonObject erLookup(final JsonObject bodyJson, Boolean failIfMultipleFou
 
                     // prepare ER Query
                     String Query = ERLookupController.getQueryfromLookup(lookupLinks.get(i).getAsJsonObject());
-                    String url = erlookupConfig.getErURL() + Query;
+                    String url = erlookupConfig.getErURL() + Query + String.format("&shallow=%s&pageSize=%d", !lookupInExternalERs, lookupLimit);
```

Comment:
Do we have any automated integration tests for this query to ER, or is it only tested manually? I can only find unittests which mocks ER responses back.

Reply:
Doesn't have any automated integration test for this feature,
for automated integration tests we have STS7 job
code we have only unit tests..

Reply:
STS7 should not be referrred to on GitHub since no-one outside Ericsson would know about it....
As long as this feature is tested properly, it's good.

Reply:
In a best world, all test should be available in RemRem code, so if other users contribute to code these functionalities, same test can be executed by other users. It would be very good if these tests can be executed in Travis-CI. For the moment EventRepository source code has not been open-sourced yet, but since EI is using the EventRepository Docker image from DockerHub already, then I guess RemRem tests can use that Docker image as well for tests.

Reply:
As tobiasake said i will create a another issue for implementing integrations tests, same as which are implemented in Eiffel-Intelligence integrations tests is using the EventRepository Docker image from DockerHub

Reply:
Sounds like a good plan!

Reply:
We will handle the integration tests in below issue. hence closing the issue.
https://github.com/eiffel-community/eiffel-remrem-generate/issues/162